commandline/src/main/java/gov/cms/qpp/conversion/CommandLineRunner.java

```diff
@@ -0,0 +1,194 @@
+package gov.cms.qpp.conversion;
+
+import gov.cms.qpp.conversion.segmentation.QrdaScope;
+import gov.cms.qpp.conversion.util.Finder;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.io.UncheckedIOException;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.apache.commons.cli.CommandLine;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CommandLineRunner implements Runnable {
+
+	private static final Logger DEV_LOG = LoggerFactory.getLogger(CommandLineMain.class);
+	private static final Pattern LITERAL_COMMA = Pattern.compile(",", Pattern.LITERAL);
+	private static final Pattern NORMAL_PATH = Pattern.compile("[a-zA-Z0-9_\\-\\s,.\\/]+");
+	private static final Pattern GLOB_FINDER = Pattern.compile("(" + NORMAL_PATH.pattern() + ").+");
+
+	private final CommandLine commandLine;
+	private final FileSystem fileSystem;
+	private Set<QrdaScope> scope;
+	private boolean doValidation;
+	private boolean doDefaults;
+	private boolean historical;
+
+	public CommandLineRunner(CommandLine commandLine) {
+		this(commandLine, FileSystems.getDefault());
+	}
+
+	public CommandLineRunner(CommandLine commandLine, FileSystem fileSystem) {
+		Objects.requireNonNull(commandLine, "commandLine");
+		Objects.requireNonNull(fileSystem, "fileSystem");
+
+		this.commandLine = commandLine;
+		this.fileSystem = fileSystem;
+	}
+
+	@Override
+	public void run() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1142)